ui/components/multichain/account-picker/account-picker.stories.js

```diff
@@ -55,3 +61,23 @@ export const ChaosWithAddressStory = (args) => (
 );
 ChaosWithAddressStory.storyName = 'Chaos with Address';
 ChaosWithAddressStory.args = { name: CHAOS_ACCOUNT.name, showAddress: true };
+
+export const CustomAccountPicker = (args) => (
+  <AccountPicker
+    {...args}
+    style={{ height: 'auto' }} // add via a custom className
+    showAddress
+    borderRadius={BorderRadius.MD}
+    borderColor={BorderColor.borderDefault}
+    paddingTop={3}
+    paddingBottom={3}
+    labelProps={{ textAlign: TextAlign.Left }}
+    block
+    endIconProps={{
+      marginLeft: 'auto',
+    }}
+    textProps={{
+      gap: 2,
+    }}
+  />
+);
```

Comment:
@NidhiKJha this is just an example you could possibly create a new component based on this